simde/simde-common.h

```diff
@@ -693,6 +693,7 @@ typedef SIMDE_FLOAT64_TYPE simde_float64;
 #    if defined(SIMDE_ARCH_AARCH64)
 #      define SIMDE_BUG_CLANG_45541
 #    endif
+#    define SIMDE_BUG_CLANG_45959
```

Comment:
Seems that `SIMDE_BUG_CLANG_45959` is not used in this PR?

Reply:
`HEDLEY_DIAGNOSTIC_PUSH
        #if defined(SIMDE_BUG_CLANG_45959)
          #pragma clang diagnostic ignored "-Wsign-conversion"
        #endif
        SIMDE_VECTORIZE_REDUCTION(&:int_res_1)
        for(int j = 0 ; j <= (upper_bound-i) ; j++){
          int_res_1 &=  (((bool_res_.i8[k] >> j) & 1 ) << i) ;
          k += 1;
        }
        HEDLEY_DIAGNOSTIC_POP`
It is used in this part of code, is something wrong here?

Reply:
my mistake! I did a search at https://github.com/nemequ/simde/pull/295/files but maybe the whole page hadn't loaded